central/policy/service/validator.go

```diff
@@ -218,14 +218,15 @@ func (s *policyValidator) getCaps(policy *storage.Policy, capsTypes string) []*s
 }
 
 func (s *policyValidator) validateCapabilities(policy *storage.Policy) error {
-	set := mapset.NewSet()
+	values := set.NewSet[string]()
 	for _, s := range s.getCaps(policy, fieldnames.AddCaps) {
```

Comment:
want to have `s.getCaps` return a `set.Set[*storage.PolicyValue]` and do `(*set).Intersect`?

Reply:
I thought about this (or rather, doing something directly set-based), but the downside is that the error message then becomes non-deterministic, or requires sorting. The chosen approach of de-duplicating but relying on the existing ordering seems more elegant to me.